src/main/connect/mod.rs

```diff
@@ -41,6 +42,7 @@ pub struct Connect<UID: Uid> {
     event_tx: crate::CrustEventSender<UID>,
     our_pk: PublicEncryptKey,
     config: CrustConfig,
+    our_addrs: HashSet<SocketAddr>,
```

Comment:
call these vars `our_direct_listeners` pls to make it less confusing to what it is.